package.json

```diff
@@ -7,6 +7,10 @@
 		"npm": ">=8.19.0"
 	},
 	"dependencies": {
+		"@emotion/react": "^11.13.3",
+		"@emotion/styled": "^11.13.0",
+		"@mui/icons-material": "^6.1.1",
+		"@mui/material": "^6.1.1",
```

Comment:
Warning that you may have a merge conflict since we're seeing other open PRs changing this file.

No action needed, just be aware.